echo2.cc

```diff
@@ -8,12 +8,12 @@
 namespace Envoy {
 namespace Filter {
 
-Network::FilterStatus Echo2::onData(Buffer::Instance& data) {
+Network::FilterStatus Echo2::onData(Buffer::Instance& data, bool) {
```

Comment:
FYI, not sure if you were on the community call on Tue, but this kind of filter breaking interface change came up (in the context of @alyssawilk's 100-continue) work. We'd like to move to a model where we can phase in changes to the filter interface without breaking all users, but for now we allow this still.

Reply:
No, I missed the meeting on tuesday.  I'll keep that in mind for the future.